release_notes/42-release-notes.adoc

```diff
@@ -423,6 +424,22 @@ Red Hat has removed the *Use Container IAM role* option from {product-title-mana
 * A sensor panic could occur in version 4.2.0 if a cluster contained deployments with an invalid image reference, for example, `image: " "`, and delegated scanning was enabled for all registries. This issue has been fixed.
 * The minimum permissions required to display 6 of the sidebar links in the {product-title-short} web portal were too strict in the 4.2.0 release, and are reduced in this release.
 
+[id="resolved-in-version-4202_{context}"]
+=== Resolved in version 4.2.2
+
+*Release date:* 24 October 2023
+
+This release of {product-title-short} fixes the following security vulnerabilities:
+
+* link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487] and link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325]: Flaw in handling multiplexed streams in the HTTP/2 protocol
+* Various CVEs in containers, including link:https://access.redhat.com/security/cve/cve-2023-4527[CVE-2023-4527], link:https://access.redhat.com/security/cve/cve-2023-4806[CVE-2023-4806], link:https://access.redhat.com/security/cve/cve-2023-4813[CVE-2023-4813], and link:https://access.redhat.com/security/cve/cve-2023-4911[CVE-2023-4911]: glibc security issues
+
+It contains the following bug fixes and changes:
+
+* Previously, {ocp} customers using the downloaded manifest bundle with automatic upgrades enabled found that sensor did not automatically upgrade, and failed with a `PRE_FLIGHT_CHECKS_FAILED` error. This issue has been fixed. (ROX-19955)
```

Comment:
is "sensor" a particular thing? Or maybe:
```suggestion
* Previously, {ocp} customers using the downloaded manifest bundle with automatic upgrades enabled found that the sensor did not automatically upgrade, and failed with a `PRE_FLIGHT_CHECKS_FAILED` error. This issue has been fixed. (ROX-19955)
```

Reply:
"sensor" here refers to a component of ACS, so it should be "Sensor"